### PR TITLE
Assert that no internal errors happen during suite write runs

### PR DESCRIPTION
**Pull request by vmalloc**

### CODE REVIEW

File: slash/core/session.py [hunk 1]
```diff
@@ -51,6 +52,12 @@ def __init__(self, reporter=None, console_stream=None):
 
         self._skip_exc_types = (exceptions.SkipTest,)
 
+    def notify_internal_error(self, error): # pylint: disable=unused-argument
```
Comment (ayalash) [inline slash/core/session.py]: Why should `error` be passed here, if it is not used? Is it for future use?

Comment (vmalloc) [reply slash/core/session.py]: Yes, although on second thought I'll drop it.

**Comment by vmalloc** [comment]
@ayalash fixed and pushed

**Comment by ayalash** [comment]
Cool :)  Merged

**Comment by vmalloc** [comment]
Thanks!

**Pull request merged**